vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewIT.java

```diff
@@ -277,6 +328,26 @@ public void gridWithDisabledSelection() {
         Assert.assertFalse(isRowSelected(grid, 1));
     }
 
+    /**
```

Comment:
Why not have the 3 new tests related to the "aria-multiselectable"/"aria-seleted" attributes next to each other in the test suite?

Reply:
I placed all 3 tests one after the other.